friends/src/App.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react';
+import {Button} from 'reactstrap';
+import Friends from './components/Friends'
+import axios from 'axios';
+import './App.css';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+        friends: [],
+        name:'',
+        age:'',
+        email:''
+    }
+}
+
+  componentDidMount() {
+      axios
+      .get("http://localhost:5000/friends")
+      .then(response => {
+          console.log(response)
+          this.setState({friends: response.data})
+      })
+      .catch( error => console.log(error))
+  }
+
+  changeHandler = (event) => {
+   this.setState({
+     [event.target.name]:event.target.value
+   })
+   
+
+  }
+
+  addFriend = (event) => {
+    event.preventDefault();
+  
+    const Friendz = {
+      id:this.state.friends.length + 1,
```

Comment:
There is no need to set a value for the ID, since it is created automatically on the server